content/blog/en/introduction-to-testing-with-dojo.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Introduction to testing with Dojo
+date: 2020-02-03
+author: Anthony Gubler & Rene Rubalcava
+---
+
+![Dojo CLI](assets/blog/introduction-to-testing-with-dojo/featured.svg)
+
+<!-- more -->
+
+With Dojo, in additional to the standard features needed to build a multitude of applications, Dojo also comes with a complete testing module to help ensure that testing your application is as painless as possible. Having a well tested application is an important consideration when building a product that you need to ship to users for a variety of reasons. A well tested application will reduce negative impact on your users and increases confidence that adding new features won't break something else. Good unit tests also help your codebase to be understood by engineers that need to make changes in the future, so be nice to your fellow engineers.
+
+Of course there are a number of different types of testing that help to ensure that an application has fewer bugs but for the purpose of this blog we'll be concentrating on unit testing.
```

Comment:
```suggestion
there are a number of different types of testing that help to ensure that an application has fewer bugs. Here we'll be concentrating on unit testing.
```

Reply:
Should this be a capital `T` for `there` now?

Reply:
Yes, thanks